src/test/testdata/debitum.csv

```diff
@@ -1,5 +1,4 @@
 Date,Transaction ID,Asset ID,Transaction Type,Turnover
 2020-08-25,405eea2a-7745-4588-8f08-5c1512987324,NA,DEPOSIT,121.91
-2020-08-25,6e71160a-d8f6-4063-b710-ded06d7599bb,6c4a6d93-faea-4d96-856c-7cdd3fb3023b,INVESTMENT,-10
 2020-09-07,b9da7662-de61-43d1-a179-c300d5695587,6c4a6d93-faea-4d96-856c-7cdd3fb3023b,REPAYMENT,10.03
```

Comment:
As I understand it, you only want to add the actual interest paid out added here and not the invested amount + interest. For testing purposes, this is OK.

Reply:
Yes, this is what my earlier comments were all about.

Currently one needs to do some preprocessing of the csv before parsing it into the P2P-parser. Debitum told me that they want to change this but no-one knows when this will happen.